nextjs-13/src/app/send/page.tsx

```diff
@@ -0,0 +1,116 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import { ClipLoader } from 'react-spinners';
+
+import MultiSelectWrapper from '@components/MultiSelectWrapper';
+import requestManager from 'lib/request/manager';
+import makeToast from 'lib/toast/makeToast';
+
+const SendNewsletter = () => {
+  const [selected, setSelected] = useState([]);
+  const [records, setRecords] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [email, setEmail] = useState('');
+
+  const getNewletters = async () => {
+    setIsLoading(true);
+
+    try {
+      const response = await requestManager('GET', 'v1/newsletter');
+
+      setIsLoading(false);
+      setRecords(response.records);
+    } catch (error) {
+      makeToast(error.message, 'error');
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    getNewletters();
+  }, []);
+
+  const afterSubmit = () => {
+    setIsLoading(false);
+    setSelected([]);
+    setEmail('');
+    getNewletters();
+  };
+
+  const handleSubmit = async (event) => {
```

Comment:
What do you think of naming this:

```suggestion
  const resetState = async (event) => {
```

Reply:
Yes I think that is a less vague name 😄

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/123/commits/126912f87b99d22d6544794aba9649f540bae17a)